tests/resources/test_security_group_egress.py

```diff
@@ -18,13 +18,13 @@
 
 
 @pytest.fixture()
-def security_group_egress_1():
+def security_group_egress_ipv4_1():
     return SecurityGroupEgress(
         **{
             "Type": "AWS::EC2::SecurityGroupEgress",
             "Properties": {
                 "GroupId": "sg-12341234",
-                "CidrIp": "1.1.1.1/0",
```

Comment:
why this change?

Reply:
`1.1.1.1/0` is not a valid IP address in this new model:

```
E           pydantic.error_wrappers.ValidationError: 2 validation errors for SecurityGroupIngress
E           Properties -> CidrIp
E             value is not a valid IPv4 network (type=value_error.ipv4network)
E           Properties -> CidrIp
E             value is not a valid dict (type=type_error.dict)
```